### ISSUE TITLE
Anaconda Prompt doesn't work in conda 4.4.7 after 'conda update --all'

### ISSUE THREAD

**Issue by Zhmoll** [conda/conda#6770]
**I'm submitting a...**
  - [x] bug report
  - [ ] feature request

## Current Behavior

Anaconda Prompt in windows doesn't work after an latest update. It showes errors below.
![default](https://user-images.githubusercontent.com/11899660/35289944-bd9b33a4-00a3-11e8-930c-09f9b2950e02.png)

### Steps to Reproduce

I just click the 'Anaconda Prompt' in the start menu. It has worked well until my latest update by excute 'conda update --all'.

I guess that it's a LF & CRLF problem in the windows but I could not make sure. I has test for several times and finally found code excute to here correctly as the picture below.
![default](https://user-images.githubusercontent.com/11899660/35289975-cea65a02-00a3-11e8-8495-96f78f8027dd.png)

Actually it turns out running this.
![default](https://user-images.githubusercontent.com/11899660/35289308-ecff1586-00a1-11e8-8274-001bfa4bb5e1.png)

See 'LF' bottom right. When I choose CLRF it works without errors.

Does conda.exe produce the file with wrong line break? And sorry for my poor English.

## Expected Behavior

It should work as well as before.

## Environment Information
<details open><summary><code>`conda info`</code></summary><p>

```
PS D:\Anaconda3\Scripts> ./conda info

     active environment : None
       user config file : C:\Users\xxx\.condarc
 populated config files : C:\Users\xxx\.condarc
          conda version : 4.4.7
    conda-build version : 3.2.2
         python version : 3.6.4.final.0
       base environment : D:\Anaconda3  (writable)
           channel URLs : https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main/win-64
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main/noarch
                          https://repo.continuum.io/pkgs/main/win-64
                          https://repo.continuum.io/pkgs/main/noarch
                          https://repo.continuum.io/pkgs/free/win-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/win-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/win-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://repo.continuum.io/pkgs/msys2/win-64
                          https://repo.continuum.io/pkgs/msys2/noarch
          package cache : D:\Anaconda3\pkgs
                          C:\Users\xxx\AppData\Local\conda\conda\pkgs
       envs directories : D:\Anaconda3\envs
                          C:\Users\xxx\AppData\Local\conda\conda\envs
                          C:\Users\xxx\.conda\envs
               platform : win-64
             user-agent : conda/4.4.7 requests/2.18.4 CPython/3.6.4 Windows/10 Windows/10.0.16299
          administrator : False
             netrc file : None
           offline mode : False
```
</p></details>

<details open><summary><code>`conda config --show-sources`</code></summary><p>

```
==> C:\Users\xxx\.condarc <==
ssl_verify: True
channels:
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main/
  - defaults
show_channel_urls: True
```
</p></details>

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>

```
# packages in environment at D:\Anaconda3:
#
_ipyw_jlab_nb_ext_conf    0.1.0            py36he6757f0_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
alabaster                 0.7.10           py36hcd07829_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
anaconda                  custom           py36h363777c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
anaconda-client           1.6.6            py36ha174c20_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
anaconda-navigator        1.6.12           py36hdad2993_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
anaconda-project          0.8.2            py36hfad2e28_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
asn1crypto                0.24.0                   py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
astroid                   1.6.0                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
astropy                   2.0.3            py36hfa6e2cd_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
attrs                     17.3.0           py36hc87868e_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
babel                     2.5.3                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
backports                 1.0              py36h81696a8_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
backports.shutil_get_terminal_size 1.0.0            py36h79ab834_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
beautifulsoup4            4.6.0            py36hd4cc5e8_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
bitarray                  0.8.1            py36h6af124b_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
bkcharts                  0.2              py36h7e685f7_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
blaze                     0.11.3           py36h8a29ca5_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
bleach                    2.1.2                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
bokeh                     0.12.13          py36h047fa9f_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
boto                      2.48.0           py36h1a776d2_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
bottleneck                1.2.1            py36hd119dfa_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
bzip2                     1.0.6                haa5b126_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ca-certificates           2017.08.26           h94faf87_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cachecontrol              0.12.3           py36hfe50d7b_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
certifi                   2017.11.5        py36hb8ac631_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cffi                      1.11.4           py36hfa6e2cd_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
chardet                   3.0.4            py36h420ce6e_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
click                     6.7              py36hec8c647_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cloudpickle               0.5.2            py36h6b1d831_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
clyent                    1.2.2            py36hb10d595_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
colorama                  0.3.9            py36h029ae33_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
comtypes                  1.1.4                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
conda                     4.4.7                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
conda-build               3.2.2                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
conda-env                 2.6.0                h36134e3_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
conda-verify              2.0.0            py36h065de53_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
console_shortcut          0.1.1                h6bb2dd7_3    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
contextlib2               0.5.5            py36he5d52c0_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
coverage                  4.4.2            py36hd32d05b_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cryptography              2.1.4            py36he1d7878_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
curl                      7.55.1               h3b839b5_4    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cycler                    0.10.0           py36h009560c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cython                    0.27.3           py36h22f4c84_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
cytoolz                   0.9.0            py36hfa6e2cd_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
dask                      0.16.1                   py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
dask-core                 0.16.1                   py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
datashape                 0.5.4            py36h5770b85_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
decorator                 4.2.1                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
distlib                   0.2.5            py36h51371be_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
distributed               1.20.2                   py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
docutils                  0.14             py36h6012d8f_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
entrypoints               0.2.3            py36hfd66bb0_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
et_xmlfile                1.0.1            py36h3d2d736_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
fastcache                 1.0.2            py36hffdae1b_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
filelock                  2.0.13           py36h20000bf_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
flask                     0.12.2           py36h98b5e8f_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
flask-cors                3.0.3            py36h8a3855d_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
freetype                  2.8                  h51f8f2c_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
get_terminal_size         1.0.0                h38e98db_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
gevent                    1.2.2            py36h342a76c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
glob2                     0.6              py36hdf76b57_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
greenlet                  0.4.12           py36ha00ad21_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
h5py                      2.7.1            py36he54a1c3_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
hdf5                      1.10.1               h98b8871_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
heapdict                  1.0.0            py36h21fa5f4_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
html5lib                  1.0.1            py36h047fa9f_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
hypothesis                3.38.5           py36h987501d_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
icc_rt                    2017.0.4             h97af966_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
icu                       58.2                 ha66f8fd_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
idna                      2.6              py36h148d497_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
imageio                   2.2.0            py36had6c2d2_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
imagesize                 0.7.1            py36he29f638_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
intel-openmp              2018.0.0             hd92c6cd_8    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ipykernel                 4.7.0            py36h2f9c1c0_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ipython                   6.2.1            py36h9cf0123_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ipython_genutils          0.2.0            py36h3c5d0ee_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ipywidgets                7.1.0                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
isort                     4.2.15           py36h6198cc5_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
itsdangerous              0.24             py36hb6c5a24_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jdcal                     1.3              py36h64a5255_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jedi                      0.11.1                   py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jinja2                    2.10             py36h292fed1_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jpeg                      9b                   hb83a4c4_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jsonschema                2.6.0            py36h7636477_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jupyter                   1.0.0            py36h422fd7e_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jupyter_client            5.2.1                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jupyter_console           5.2.0            py36h6d89b47_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jupyter_core              4.4.0            py36h56e9d50_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jupyterlab                0.31.1                   py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
jupyterlab_launcher       0.10.2                   py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
lazy-object-proxy         1.3.1            py36hd1c21d2_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libiconv                  1.15                 h1df5818_7    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libpng                    1.6.32               h140d38e_4    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libssh2                   1.8.0                h75d9407_3    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libtiff                   4.0.9                h0f13578_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libxml2                   2.9.4                h41ea7b2_6    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
libxslt                   1.1.29               h0037b19_6    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
llvmlite                  0.21.0           py36he0b0552_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
locket                    0.2.0            py36hfed976d_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
lockfile                  0.12.2           py36h0468280_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
lxml                      4.1.1            py36he0adb16_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
lzo                       2.10                 h6df0209_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
markupsafe                1.0              py36h0e26971_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
matplotlib                2.1.1            py36h2062329_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mccabe                    0.6.1            py36hb41005a_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
menuinst                  1.4.10           py36h42196fb_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mistune                   0.8.3                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mkl                       2018.0.1             h2108138_4    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mkl-service               1.1.2            py36h57e144c_4    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
mpmath                    1.0.0            py36hacc8adf_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
msgpack-python            0.5.1            py36he980bc4_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
multipledispatch          0.4.9            py36he44c36e_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
navigator-updater         0.1.0            py36h8a7b86b_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
nbconvert                 5.3.1            py36h8dc0fde_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
nbformat                  4.4.0            py36h3a5bc1b_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
networkx                  2.0              py36hff991e3_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
nltk                      3.2.5            py36h76d52bb_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
nose                      1.3.7            py36h1c3779e_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
notebook                  5.3.1                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
numba                     0.36.2          np113py36h0128aa7_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
numexpr                   2.6.4            py36h30784b8_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
numpy                     1.13.3           py36ha320f96_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
numpydoc                  0.7.0            py36ha25429e_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
odo                       0.5.1            py36h7560279_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
olefile                   0.44             py36h0a7bdd2_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
openpyxl                  2.4.9            py36h6c6577a_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
openssl                   1.0.2n               h74b6da3_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
packaging                 16.8             py36ha0986f6_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pandas                    0.22.0           py36h6538335_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pandoc                    1.19.2.1             hb2460c7_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pandocfilters             1.4.2            py36h3ef6317_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
parso                     0.1.1            py36hae3edee_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
partd                     0.3.8            py36hc8e763b_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
path.py                   10.5             py36h2b94a8f_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pathlib2                  2.3.0            py36h7bfb78b_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
patsy                     0.5.0                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pep8                      1.7.1                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pickleshare               0.7.4            py36h9de030f_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pillow                    5.0.0            py36h0738816_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pip                       9.0.1            py36h226ae91_4    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pkginfo                   1.4.1            py36hb0f9cfa_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pluggy                    0.6.0            py36hc7daf1e_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ply                       3.10             py36h1211beb_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
progress                  1.3              py36hbeca8d3_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
prompt_toolkit            1.0.15           py36h60b8f86_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
psutil                    5.4.3            py36hfa6e2cd_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
py                        1.5.2            py36hbcfbabc_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pycodestyle               2.3.1            py36h7cc55cd_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pycosat                   0.6.3            py36h413d8a4_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pycparser                 2.18             py36hd053e01_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pycrypto                  2.6.1            py36he68e6e2_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pycurl                    7.43.0.1         py36h74b6da3_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pyflakes                  1.6.0            py36h0b975d6_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pygments                  2.2.0            py36hb010967_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pylint                    1.8.1                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pympler                   0.5              py36h863af32_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pyodbc                    4.0.21           py36h82c9817_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pyopenssl                 17.5.0           py36h5b7d817_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pyparsing                 2.2.0            py36h785a196_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pyqt                      5.6.0            py36hb5ed885_5    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pysocks                   1.6.7            py36h698d350_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pytables                  3.4.2            py36h71138e3_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pytest                    3.3.2                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
python                    3.6.4                h6538335_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
python-dateutil           2.6.1            py36h509ddcb_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pytz                      2017.3           py36h1d3fa6b_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pywavelets                0.5.2            py36hc649158_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pywin32                   221              py36h9c10281_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pyyaml                    3.12             py36h1d1928f_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
pyzmq                     16.0.3           py36he714bf5_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
qt                        5.6.2           vc14h6f8c307_12  [vc14]  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
qtawesome                 0.4.4            py36h5aa48f6_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
qtconsole                 4.3.1            py36h99a29a9_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
qtpy                      1.3.1            py36hb8717c5_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
requests                  2.18.4           py36h4371aae_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
rope                      0.10.7           py36had63a69_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
ruamel_yaml               0.15.35          py36hfa6e2cd_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
scikit-image              0.13.1           py36hfa6e2cd_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
scikit-learn              0.19.1           py36h53aea1b_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
scipy                     1.0.0            py36h1260518_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
seaborn                   0.8.1            py36h9b69545_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
send2trash                1.4.2                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
setuptools                38.4.0                   py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
simplegeneric             0.8.1            py36heab741f_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
singledispatch            3.4.0.3          py36h17d0c80_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
sip                       4.18.1           py36h9c25514_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
six                       1.11.0           py36h4db2310_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
snowballstemmer           1.2.1            py36h763602f_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
sortedcollections         0.5.3            py36hbefa0ab_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
sortedcontainers          1.5.9                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
sphinx                    1.6.6                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
sphinxcontrib             1.0              py36hbbac3d2_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
sphinxcontrib-websupport  1.0.1            py36hb5e5916_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
spyder                    3.2.6                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
sqlalchemy                1.2.1            py36hfa6e2cd_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
sqlite                    3.20.1               h9eeafa9_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
statsmodels               0.8.0            py36h6189b4c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
sympy                     1.1.1            py36h96708e0_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
tblib                     1.3.2            py36h30f5020_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
testpath                  0.3.1            py36h2698cfe_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
tk                        8.6.7                hcb92d03_3    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
toolz                     0.9.0                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
tornado                   4.5.3                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
traitlets                 4.3.2            py36h096827d_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
typing                    3.6.2            py36hb035bda_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
unicodecsv                0.14.1           py36h6450c06_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
urllib3                   1.22             py36h276f60a_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
vc                        14                   h0510ff6_3    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
vs2015_runtime            14.0.25123                    3    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
wcwidth                   0.1.7            py36h3d5aa90_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
webencodings              0.5.1            py36h67c50ae_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
werkzeug                  0.14.1                   py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
wheel                     0.30.0           py36h6c3ec14_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
widgetsnbextension        3.1.0                    py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
win_inet_pton             1.0.1            py36he67d7fd_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
win_unicode_console       0.5              py36hcdbd4b5_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
wincertstore              0.2              py36h7fe50ca_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
wrapt                     1.10.11          py36he5f5981_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
xlrd                      1.1.0            py36h1cb58dc_1    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
xlsxwriter                1.0.2            py36hf723b7d_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
xlwings                   0.11.5                   py36_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
xlwt                      1.3.0            py36h1a4751e_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
yaml                      0.1.7                hc54c509_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
zict                      0.1.3            py36h2d8e73e_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
zlib                      1.2.11               h8395fce_2    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
zope                      1.0                      py36_0    defaults
zope.interface            4.4.3            py36h56cab3c_0    https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
```
</p></details>

**Comment by kalefranz** [comment]
I thought python was supposed to take care of CRLF for us on windows in most cases.  Possible that's not the case, but then confused how more people haven't reported this issue so far.  Looking into it though...

**Comment by kalefranz** [comment]
For reference, if we end up making a modification, it may be in this line:

    https://github.com/conda/conda/blob/4.4.7/conda/activate.py#L90

**Comment by Zhmoll** [comment]
The bat file generated in temp directory contains only LF. When I change '\n' into '\r\n' at https://github.com/conda/conda/blob/4.4.7/conda/activate.py#L90, the prompt works! But I'm still not sure because others haven't reported this issue.

**Comment by kalefranz** [comment]
Thank you for the help is identifying and isolating the root cause of your problem.  Fixes always come much faster that way!  #6775 will need further review, but I'm guessing it will be included in the next patch release.
